datasets.py

```diff
@@ -944,13 +944,27 @@ def __init__(self, transform):
         # directory storing gps information
         self.gps_folder = '/Users/home/Desktop/research/data/cityscapes/vehicle_trainvaltest/vehicle/train'
 
-        # boundary shapefile
+        # local boundary shapefile from 
+        # https://maps.princeton.edu/catalog/stanford-nh891yz3147
```

Comment:
not sure where would make the most sense, but somewhere in the dataset class as well as README can you make super clear the two paths that classes with geography labels can progress in (e.g., region label vs gps)? and how the user can specify that?

Reply:
added a mention in the readme